### ISSUE TITLE
Dark theme doesn't seem to affect Vivaldi

### ISSUE THREAD

**Issue by WetGeek** [getsolus/packages#1099]
### Summary

This is the theme that I'm using.  I think it's the default, because I don't remember choosing it, but it's what I would have chosen regardless.  My eyes are rather ensitive to bright light.
![image](https://github.com/getsolus/packages/assets/6805994/06d0d5f6-78a6-48ea-ad40-e55fb409e65f)

The theme apparently applies to the applications that are part of the Xfce DE, but not to one that I installed from the repository--in particular, Vivaldi.  Opening any of the many sites that it hosts provides a painful blast of white light.
![image](https://github.com/getsolus/packages/assets/6805994/ff4912f3-9d56-4140-a4c0-d894d84d1707)

### Steps to reproduce

Just launch Vivaldi (on hardware) and you should notice right away that it is not using any kind of dark theme.

### Expected result

I expected web pages to be displayed in a dark theme, as they are on a fully-updated Xfce RC2 VM that runs on my daily-driver laptop.  (See similar image in comment below.)

### Actual result

This should be dark as well (Xfce on bare metal) but it's apparently unaffected by the theme.  I have no idea why the hardware version differs from the virtual version, but as I pointed out, it's painful to look at.

### Environment

- [X] Is system up to date?

### Repo

Shannon (stable)

### Desktop Environment

Xfce

### System details

![image](https://github.com/getsolus/packages/assets/6805994/a4491963-7e03-4e81-a8c1-3aa2b7799bad)

**Comment by WetGeek** [comment]
In comparison, this is a screenshot of the same full-screen image, taken from Vivaldi running on an Xfce VM that's fully updated, the same as the image above.
![image](https://github.com/getsolus/packages/assets/6805994/0a9e2cc6-9908-4878-8518-db77b09ce64f)

**Comment by TraceyC77** [comment]
I tried to reproduce this on a fresh XFCE 4.5 RC5 installation on a VMM VM. There was no customization.
The problem is confirmed and is specific to XFCE.

## Reproduction Steps:
Booted into the XFCE VM after installation. The theme that was set by default was the same one in the report (Qogir-Dark).
Checked for updates, none were available.
Vivaldi is not installed by default, installed vivaldi-stable through Software Center.
Launched Vivaldi. Went through the setup including "Choose your style". Note: The default style "Vivaldi" is white with a red bar on top. The "Dark" style matches the screenshots, having a dark bar and tabs, so I chose that.
Opened this issue in Vivaldi on the VM.

## Result
Confirmed that Vivaldi displays a white background. In contrast, Firefox displays a dark theme.
This was true for other pages as well, such as duckduckgo.com.



## Results in Other DEs
In contrast, both Firefox and Vivaldi display these pages with a dark background / theme for me on Plasma unstable.
Tested on a fresh Budgie VM with 4.5 RC5, also using the default theme (which is dark), it displays pages correctly with a dark background.

**Comment by EbonJaeger** [comment]
I've discovered that switching to another theme and then back to Qogir-Dark makes Vivaldi correctly use dark mode on websites. This leads me to believe that there is a setting somewhere that we're not setting by default, but should be. I asked in the XFCE Matrix channel about what it might be, but they haven't gotten back to me yet.

**Comment by TraceyC77** [comment]
Ping @EbonJaeger - we should probably sort this for 4.8. Did the XFCE people ever get back to you?

**Comment by EbonJaeger** [comment]
Not really. If I remember correctly, Xfce has a handler that sets prefer dark on theme changes, but it isn't called on startup, leading to the incorrect default being used.
